includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -2197,11 +2266,17 @@ public static function print_status_meta_box( $post ) {
 	 * @return void
 	 */
 	public static function print_stylesheets_meta_box( $post ) {
-		$stylesheets = get_post_meta( $post->ID, '_amp_stylesheets', true );
+		$stylesheets = get_post_meta( $post->ID, self::STYLESHEETS_POST_META_KEY, true );
 		if ( empty( $stylesheets ) ) {
 			printf(
 				'<p><em>%s</em></p>',
-				esc_html__( 'No stylesheet data available. Please try re-checking this URL.', 'amp' )
+				wp_kses_post(
+					sprintf(
+						/* translators: placeholder is URL to recheck the post */
+						__( 'Stylesheet information for this URL is no longer available. Such data is automatically deleted after a week to reduce database storage. It is of little value to store long-term given that it becomes stale as themes and plugins are updated. To obtain the latest stylesheet information, <a href="%s">recheck this URL</a>.', 'amp' ),
```

Comment:
I suppose the non-breaking space in here is not intentional.

![image](https://user-images.githubusercontent.com/16200219/89373960-95575f80-d6d9-11ea-9bd4-71bef6833388.png)


```suggestion
						__( 'Stylesheet information for this URL is no longer available. Such data is automatically deleted after a week to reduce database storage. It is of little value to store long-term given that it becomes stale as themes and plugins are updated. To obtain the latest stylesheet information, <a href="%s">recheck this URL</a>.', 'amp' ),
```